src/panel/category/CategoryPanel.jsx

```diff
@@ -0,0 +1,227 @@
+/* globals _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Panel from 'src/components/ui/Panel';
+import PoiCategoryItems from './PoiCategoryItems';
+import debounce from 'src/libs/debounce';
+import IdunnPoi from 'src/adapters/poi/idunn_poi';
+import nconf from '@qwant/nconf-getter';
+import Telemetry from 'src/libs/telemetry';
+import CategoryService from 'src/adapters/category_service';
+import SearchInput from 'src/ui_components/search_input';
+import layouts from 'src/panel/layouts.js';
+import { sources } from 'config/constants.yml';
+
+const MAX_PLACES = Number(nconf.get().category.maxPlaces);
+
+export default class CategoryPanel extends React.Component {
+  static propTypes = {
+    categoryName: PropTypes.string.isRequired,
+    query: PropTypes.string,
+    bbox: PropTypes.string,
+  }
+
+  static defaultProps = {
+    query: '',
+    bbox: '',
+  }
+
+  state = {
+    pois: [],
+    dataSource: null,
+    isLoading: true,
+  }
+
+  componentDidMount() {
+    const { name, label } = CategoryService.getCategoryByName(this.props.categoryName);
+    Telemetry.add(Telemetry.POI_CATEGORY_OPEN, null, null, { category: name });
+    SearchInput.setInputValue(label.charAt(0).toUpperCase() + label.slice(1));
+
+    this.fitMap();
+    this.clickHandler = listen('click_category_poi', this.selectPoi);
+    this.mapMoveHandler = listen('map_moveend', debounce(this.refresh, 300));
+  }
+
+  componentDidUpdate() {
+    const panelContent = document.querySelector('.panel-content');
+    if (panelContent) {
+      panelContent.scrollTop = 0;
+    }
+  }
+
+  componentWillUnmount() {
+    if (!this._navToCategoryMarker) {
+      SearchInput.setInputValue('');
+      fire('remove_category_markers');
+    }
+    unListen(this.clickHandler);
```

Comment:
Removing this listener introduces a regression: it's now impossible to select another marker after a first result has been clicked (and the poi panel has replaced the category panel).